### PR TITLE
chore: setup tailwind

### PR DESCRIPTION
**Pull request by claudio-OST**
Setup and initial configuration for Tailwind

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** @type { import('@storybook/react').Preview } */
 import { withThemeByClassName } from '@storybook/addon-styling';
 
+/* TODO: update import to your tailwind styles file. If you're using Angular, inject this through your angular.json config instead */
```
Comment (malinovic) [inline .storybook/preview.js:4]: Remove this comment and also the comment on line 7

File: .storybook/preview.js [hunk 2]
```diff
@@ -16,6 +19,14 @@ const preview = {
   decorators: [
```
Comment (malinovic) [inline .storybook/preview.js:19]: Remove theming decorators

Comment (claudio-OST) [reply .storybook/preview.js]: @malinovic fixed

File: .storybook/preview.js (hunk 1 above)
Comment (claudio-OST) [reply .storybook/preview.js]: @malinovic fixed

**Comment by malinovic** [comment]
Hello @claudio-OST, all pull request comments are resolved. I will merge the pull request.

**Pull request merged**